### DISCUSSION TITLE
L'architecture Angular parfaite ? | Bonjour Angular

### DISCUSSION THREAD

**Discussion by giscus** [bonjour-angular/apprendre#21] [General]
# L'architecture Angular parfaite ? | Bonjour Angular

L'architecture Angular parfaite ?

https://apprendre.bonjour-angular.com/articles/architecture-angular-parfaite/

**Comment by AJ1062910** [comment]
Mais du coup avec le mode standalone, est ce toujours utile de faire le shared ?

**Reply by KevTale** [reply]
Oui, ça n'a pas vraiment de lien. Je ne fais pas de SharedModule mais simplement un folder `shared` pour signifier que ce qui est en dessous est partagé à divers endroit de mon app.

**Comment by maequise** [comment]
Intéressant comme approche, mais étrange je la trouve moins "userfriendly" et moins DX qu'une découpe un chouilla plus brute (avec des dossiers, et des regroupements plus métier), ça doit être mon côté Java qui me borne ^^.
Mais à l'occasion je tenterai sur un projet perso pour voir :). 

En revanche, tu parles d'archi hexagonale, est-ce que ça a du sens sur un projet Angular, on même front-end de façon générale, est-ce que tu as un exemple concret en tête ?
